docs/Reference/Pantheon-CLI-Syntax.md

```diff
@@ -13,6 +13,18 @@ Runs the Pantheon Ethereum full node client.
 
 ## Options
 
+### accounts-whitelist
+
+```bash tab="Syntax"
+--accounts-whitelist[=<hex string of account public key>[,<hex string of account public key>...]...
```

Comment:
I'm not sure if the cli command needs the `[` and `]`. At least it didn't before I went on holiday :)

Reply:
It doesn't need it - it's to indicate that part is optional (ie, you can specify an empty list). Also, it matches what's in CLI help and is consistent with the other commands where an empty list can be specified :-) Will follow up with @jakehaugen and @NicolasMassart to confirm for the CLI changes.

Reply:
Yes [] indicates optional values, no need to use them in the actual CLI value